clients/ui/frontend/src/app/hooks/useNotification.ts

```diff
@@ -0,0 +1,110 @@
+import { useContext } from "react";
+import { NotificationContext } from "../context/NotificationContext";
+import { AlertVariant } from "@patternfly/react-core";
+import React from "react";
+
+enum NotificationTypes {
+  SUCCESS = 'success',
+  ERROR = 'error',
+  INFO = 'info',
+  WARNING = 'warning'
+}
+
+type NotificationProps = (title: string, message?: React.ReactNode) => void;
+
+type NotificationRemoveProps = (id: number | undefined) => void;
+
+type NotificationTypeFunc = {
+  [key in NotificationTypes]: NotificationProps
+}
+
+interface NotificationFunc extends NotificationTypeFunc {
+  remove: NotificationRemoveProps
+}
```

Comment:
This is a little hacky. Maybe we should just have a separate hook for when we want to return all notifications and do something with them like removing one.